.github/workflows/test.yml

```diff
@@ -78,6 +78,20 @@ jobs:
           cd bindings/python
           pytest -vs tests
 
+      - name: Update to dev version and build release
+        if: github.ref == 'refs/heads/main' and python-version: 3.6
```

Comment:
This line is invalid. Should be I think: 
if: github.ref == 'refs/heads/main' and  ${{ matrix.python-version == 3.6 }}